draft-ietf-taps-interface.md

```diff
@@ -1043,9 +1043,13 @@ This property allows the application to express a preference for the use of
 temporary local addresses, sometimes called "privacy" addresses {{!RFC4941}}.
 Temporary addresses are generally used to prevent linking connections over time
 when a stable address, sometimes called "permanent" address, is not needed.
-Note that if an application Requires the use of temporary addresses, the
-resulting Connection cannot use IPv4, as temporary addresses do not exist in
-IPv4.
+There are two caveats to note when specifying this property. First, if an
+application Requires the use of temporary addresses, the resulting Connection 
+cannot use IPv4, as temporary addresses do not exist in IPv4. Second, temporary
+local addresses may involve privacy/performance tradeoffs with protocol 
+stacks and networks that use session resumption in the presence of network 
```

Comment:
I don't understand what is intended by "using a stack"... "in the presence of network address translation"

Reply:
I agree with Gorry. A NAT would have a similar impact as a temporary address; proof-of-address tokens would no longer function.

Reply:
Comment from Martin: "in the presence of NAT" is wrong, should be removed